install/Knative-with-GiantSwarm.md

```diff
@@ -0,0 +1,184 @@
+# Knative Install on Giant Swarm cluster
+
+This guide walks you through the installation of the latest version of
+Knative using pre-built images.
+
+You can find [guides for other platforms here](README.md).
+
+## Before you begin
+
+Knative requires a Kubernetes cluster v1.10 or newer. `kubectl` v1.10 is also
+required.  This guide walks you through creating a cluster with the correct
+specifications for Knative on Giant Swarm managed cluster.
+
+This guide assumes you are using bash in a Mac or Linux environment; some
+commands will need to be adjusted for use in a Windows environment.
+
+### Installing the Giant Swarm CLI
+
+1. If you already have `gsctl` version `0.14.1` or later installed, you can skip to the next section and install `kubectl`
+
+Install `gsctl` by following the instructions for your operating system.
+See the [full installation instructions](https://docs.giantswarm.io/reference/gsctl/#install).
+
+#### MacOS
+
+```console
+brew tap giantswarm/giantswarm
+brew install gsctl
+```
+
+#### Linux 64-bit
+
+```console
+curl -O https://downloads.giantswarm.io/gsctl/0.14.1/gsctl-0.14.1-linux-amd64.tar.gz
+tar xzf gsctl-0.14.1-linux-amd64.tar.gz
+sudo cp gsctl-0.14.1-linux-amd64/gsctl /usr/local/bin/
+```
+
+### Installing kubectl
+
+1.  If you already have `kubectl` CLI, run `kubectl version --short` to check
+    the version. You need v1.10 or newer. If your `kubectl` is older, follow the
+    next step to install a newer version.
+
+2.  [Install the kubectl CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl).
+
+## Cluster Setup
+
+Now that we have all the tools, we need a Kubernetes cluster to install Knative.
+
+### Configure your Giant Swarm installation
+
+First let's identify your Giant Swarm installation and save it for use later.
+
+1. Run `gsctl login <user@example.com> --endpoint <api.example.com>` replacing with your email account and your installation API endpoint.
```

Comment:
I suggest moving all the information in front of the command. Something like:
Run the following command, replacing your email account and your installation API endpoint with the correct values:
`gsctl login <user@example.com> --endpoint <api.example.com>`